src/Map/Map.h

```diff
@@ -39,12 +43,19 @@ class Map
     // recalculate the owner of a node
     void updateOwner(int index);
 
-public:
     // constructors and destructors
     Map(std::vector<Node> inputNodes, std::string startRegion, std::vector<std::string> regions, std::vector<std::string> continents, std::vector<std::string> players);
     Map(Map const &m);
```

Comment:
You can get rid of the copy constructor, it doesn't actually do anything anymore.